content/2-5.md

```diff
@@ -137,114 +146,102 @@ Explain how form instructions allow people to understand the purpose of form con
-* Demonstrate use of mechanisms to provide in-line error messages for form controls, such as using the HTML element `label` or the WAI-ARIA attribute `aria-describedby`. Explain that programmatic associations help text-to-speech and mainstream technologies relate error messages to their specific field as users move around. Explain that WAI-ARIA live regions can be used to provide instant feedback as users are typing in a form control or as validation outputs appear so that people using screen readers can understand the information provided. Examples of how to code in-line error messages are provided in the WAI tutorials on [In-line feedback](https://www.w3.org/WAI/tutorials/forms/notifications/#inline).
-* Demonstrate use of some common HTML5 elements, such as `date`, `time`, `url`, `email`, or `pattern`. Explain that they allow users to enter data more reliably and help validation. Examples of how to validate common input are provided in the WAI tutorials on [Validating common input](https://www.w3.org/WAI/tutorials/forms/validation/#validating-common-input).
-
-#### Ideas to Assess Knowledge for Topic
-
-Optional ideas to support assessment.
+Optional ideas to support assessment:
 
-* Practical &mdash; Students are presented with a set of form controls and are asked to code that they are required. Assess how students make use of the HTML attribute `required` and/or some text in the label of the form control.
 * Practical &mdash; Students are presented with a set of form control errors and are asked to provide a list of messages that explain such errors. Assess students' knowledge of how to group and communicate errors in a form control.
-* Practical &mdash; Students are presented with a set of form controls with errors marked up using color alone. Then they are asked to mark up these error messages using text in the vicinity of each of the form controls. Assess students' knowledge about how to associate form errors with their corresponding form control.
-* Practical &mdash; Students are presented with a form control that outputs notifications as the user is typing to indicate the strength of a password, and are asked to code accessible notifications. Assess students' knowledge of live regions to convey notifications.
+* Practical &mdash; Students are presented with a set of form controls with errors marked up using color alone. Then they are asked to mark up these error messages using text in the vicinity of each of the form controls. Assess students' knowledge of how to associate form errors with their corresponding form control.
+* Practical &mdash; Students are presented with a form submission containing errors. Then they are asked to provide known corrections for the errors if they exist, and to associate them appropriately with each of the form controls. Assess students' understanding of how to provide suggestions for error correction.
```

Comment:
This seems to suggest that the student is to correct the information put into a form rather than the form itself.

Reply:
It now reads:

> * Practical &mdash; Students are presented with a form submission containing errors. Then they are asked to associate known corrections that may exist for the errors with each of the form controls. Assess students' understanding of how to associate known corrections with their corresponding form controls.